### PR TITLE
Workaround Description for Kibana Import Bug 1895

### PR DESCRIPTION
**Pull request by GoebelL**
* Workaround description for bug 1895

### CODE REVIEW

File: Markdown/cmm-2.0.14/release-notes/2-restrictions.md [hunk 1]
```diff
@@ -60,4 +60,16 @@ Errors occur due to this known issue in Kibana 7.3.0 [Kibana Issue 41567](https:
...
   ```
+
+### Importing Objects in Kibana
+Exported JSON files can only be imported with API, due to a known issue with Kibana 7.3.0:  
+When attempting to import from web UI, an error message is shown, including information of "Unsupported Media Type".
+
+* Workaround:  
+  Use following `curl` command to import the JSON data from API: 
+  ```
+  curl –XPOST “<ip-address of kibana server>:5601/api/saved_objects/_import?overwrite=true” -H "kbn-xsrf: true" --form file=@<file to be imported>
```
Comment (chaconpiza) [inline Markdown/cmm-2.0.14/release-notes/2-restrictions.md:71]: Thanks for the PR, let's use the these kind of quotes: `"`

File: Markdown/cmm-2.0.14/system-operators/4-operation-and-maintenance.md [hunk 2]
```diff
@@ -1151,7 +1151,19 @@ to recall them. For details on saving, loading, and sharing dashboards for log m
...
 of the Monitoring Service to directly access Grafana and export dashboards. For backing up
 and restoring the exported dashboards, you can then use the standard mechanisms of your file
 system. For details on exporting monitoring dashboards, refer to the *Grafana documentation*.
+
+**Import**  
+> **Note**: Due to an existing bug in Kibana, the exported JSON file can only be imported with API. Otherwise, when attempting to import from UI, an error message is shown, including information of "Unsupported Media Type".
+
+Use following `curl` command to import the JSON data from API: 
+```
+curl –XPOST “<ip-address of kibana server>:5601/api/saved_objects/_import?overwrite=true” -H "kbn-xsrf: true" --form file=@<file to be imported>
```
Comment (chaconpiza) [inline Markdown/cmm-2.0.14/system-operators/4-operation-and-maintenance.md:1167]: Thanks for the PR, let's use the these kind of quotes: "

**Review by chaconpiza — CHANGES_REQUESTED** [review]

File: Markdown/cmm-2.0.14/release-notes/2-restrictions.md (hunk 1 above)
Comment (GoebelL) [reply Markdown/cmm-2.0.14/release-notes/2-restrictions.md]: Done

File: Markdown/cmm-2.0.14/system-operators/4-operation-and-maintenance.md (hunk 2 above)
Comment (GoebelL) [reply Markdown/cmm-2.0.14/system-operators/4-operation-and-maintenance.md]: Done

**Review by chaconpiza — APPROVED** [review]

**Pull request merged**